libs/ui/widgets/src/lib/popover/popover.component.ts

```diff
@@ -0,0 +1,82 @@
+import {
+  Component,
+  AfterViewInit,
+  ElementRef,
+  Input,
+  ViewChild,
+  OnDestroy,
+  OnChanges,
+  SimpleChanges,
+  TemplateRef,
+  Renderer2,
+  ViewContainerRef,
+  EmbeddedViewRef,
+} from '@angular/core'
+import tippy, { Instance } from 'tippy.js'
+
+@Component({
+  selector: 'gn-ui-popover',
+  templateUrl: './popover.component.html',
+  styleUrls: ['./popover.component.css'],
```

Comment:
Could you please make this component standalone? this is what we do for every new component now

Reply:
what is standalone component, and what does it apply ? i had based my code on ProgressBarComponent which is not standalone

Reply:
@jahow  I have done it yersterday, and i fixed the lint error this morning should be ok now